docs/content/sre/tenant-operator/hibernation.md

```diff
@@ -14,7 +14,7 @@ hibernation:
 
 > Note: both sleep and wake schedules must be specified for your Hibernation schedule to be valid.
 
-Additionally, adding the `hibernation.stakater.com/exclude: 'true'` annotation to a namespace, excludes that namespace from hibernating.
+Additionally, adding the `hibernation.stakater.com/exclude: 'true'` annotation to a namespace, excludes that namespace from hibernating. (This is true for hibernation applied via the Tenant Custom Resource.)
```

Comment:
This is `only` true for hibernation applied via the Tenant Custom Resource

or something like that ^